src/renderer/dx/DxRenderer.cpp

```diff
@@ -112,6 +283,134 @@ DxEngine::~DxEngine()
     return S_OK;
 }
 
+int DbgPrint(const char* format, ...)

```

Comment:
Oof. No me gusta. This can't stay around. We'll have to use the existing logging frameworks (with WIL and such. The `LOG` macros should actually emit a debug string if you're attached.